vkcs/cdn/spec.json

```diff
@@ -0,0 +1,1431 @@
+                                                    "schema_definition": "listplanmodifier.RequiresReplaceIfConfigured()"
+                                                }
+                                            }
+                                        ]
+                                    }
+                                }
+                            ]
+                        }
+                    },
+                    {
+                        "name": "preset_applied",
+                        "bool": {
+                            "computed_optional_required": "computed",
+                            "description": "Protocol used by CDN servers to request content from an origin source. "
+                        }
+                    },
+                    {
+                        "name": "secondary_hostnames",
+                        "list": {
+                            "element_type": {
+                                "string": {}
+                            },
+                            "computed_optional_required": "computed_optional",
+                            "description": "Additional delivery domains (CNAMEs) that will be used to deliver content via the CDN.",
+                            "plan_modifiers": [
+                                {
+                                    "custom": {
+                                        "imports": [
+                                            {
+                                                "path": "github.com/hashicorp/terraform-plugin-framework/resource/schema/listplanmodifier"
+                                            }
+                                        ],
+                                        "schema_definition": "listplanmodifier.RequiresReplace()"
+                                    }
+                                }
+                            ],
+                            "default": {
+                                "custom": {
+                                    "imports": [
+                                        {
+                                            "path": "github.com/hashicorp/terraform-plugin-framework/resource/schema/listdefault"
+                                        },
+                                        {
+                                            "path": "github.com/vk-cs/terraform-provider-vkcs/vkcs/internal/framework/types",
+                                            "alias": "fwtypes"
+                                        }
+                                    ],
+                                    "schema_definition": "listdefault.StaticValue(fwtypes.ListEmpty(types.StringType))"
+                                }
+                            }
+                        }
+                    },
+                    {
+                        "name": "shielded",
+                        "bool": {
+                            "computed_optional_required": "computed",
```

Comment:
we cannot set it?

Reply:
Provided support for changing shielding settings. Added data source to retrieve an available origin point of presence (POPs), so user can obtain its id to specify in the resource shielding.